test/integ/axpy/src/axpy.cpp

```diff
@@ -30,25 +30,24 @@ class AxpyKernel
     //! \tparam TAcc The type of the accelerator the kernel is executed on..
     //! \tparam TElem The matrix element type.
     //! \param acc The accelerator the kernel is executed on.
-    //! \param numElements Specifies the number of elements of the vectors X and Y.
     //! \param alpha Scalar the X vector is multiplied with.
     //! \param X Vector of at least n elements.
     //! \param Y Vector of at least n elements.
     ALPAKA_NO_HOST_ACC_WARNING
-    template<typename TAcc, typename TElem, typename TIdx>
+    template<typename TAcc, typename TMemoryHandle, typename TElem, typename TIdx>
     ALPAKA_FN_ACC auto operator()(
         TAcc const& acc,
-        TIdx const& numElements,
         TElem const& alpha,
-        TElem const* const X,
-        TElem* const Y) const -> void
+        alpaka::Accessor<TMemoryHandle, TElem, TIdx, 1, alpaka::ReadAccess> const X,
```

Comment:
I think `BufferAccessor` is a must here. Otherwise this will never work with the SYCL backend.

Reply:
I don't think `BufferAccessor` is required, just having multiple `TMemoryHandle`.